drivers/include/cc1xxx_common.h

```diff
@@ -0,0 +1,138 @@
+/*
+ * Copyright (C) 2017 Freie Universität Berlin
+ *               2018 Otto-von-Guericke-Universität Magdeburg
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    drivers_cc1xxx CC1100/CC1100e/CC1101/CC1200 common code
+ * @ingroup     drivers_netdev
+ * @ingroup     net_gnrc_netif
+ * @{
+ *
+ * @file
+ * @brief   CC110x/CC1200 adaption for @ref net_gnrc_netif
+ *
+ * @author  Martine Lenders <m.lenders@fu-berlin.de>
+ * @author  Hauke Petersen <hauke.petersen@fu-berlin.de>
+ * @author  Marian Buschsieweke <marian.buschsieweke@ovgu.de>
+ *
+ * Supported Transceivers
+ * ======================
+ *
+ * This adaption layer is written to be used by CC110x and CC1200 transceivers,
+ * but any transceiver using layer 2 addresses which are 1 byte in size would
+ * likely be able to use it. Keep in mind that both CC110x and CC1200 are able
+ * to transmit frames of up to 255 bytes (thus 253 bytes of payload, as the
+ * layer 2 header is 2 bytes in size). Other transceivers only supporting
+ * smaller frames may not be able to use all the upper layer protocols supported
+ * by the CC110x and CC1200 transceivers.
+ *
+ * Frame Format
+ * ============
+ *
+ *      0                   1                   2                   3
+ *      0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
+ *     +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+ *     |  Destination  |    Source     |  Payload...
+ *     +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+ *
+ * | Field       | Description                     |
+ * |-------------|---------------------------------|
+ * | Destination | The layer 2 destination address |
+ * | Source      | The layer 2 source address      |
+ * | Payload     | The payload (variable size)     |
+ *
+ * Please note that the payload needs to be able to carry 6LoWPAN frames.
+ *
+ * Layer 2 Broadcast
+ * =================
+ *
+ * This adaption layer assumes that the layer 2 address `0x00` is reserved for
+ * layer 2 broadcast, which is true for CC110x and CC1200 transceivers (provided
+ * they are configured accordingly). If more users of this adaption layer are
+ * added, this behaviour might needs to be more generalized.
```

Comment:
I would add some reference to `@ref CC1XXX_BCAST_ADDR` somewhere in this paragraph, but that is not a must.